public/extra_descriptions/simvi.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <div>simvi</div>
+
+  <div>from tekezo's vi, dosen't work with mac os's combination move keys</div>
+  <div>so I added keysets like below</div>
+  <div>for work with mac os's combination keys</div>
+
+  <div>option + s + h,j,k,l</div>
+  <div>shift + option + s + h,j,k,l</div>
+  <div>command + s + h,j,k,l</div>
+  <div>shift + command + s + h,j,k,l</div>
+
+  <div>highly recommended to vi + mac combination keys lover users</div>
+</div>
```

Comment:
Corrected HTML tags. Normally, you should use `p` for paragraphs.

```suggestion
<link rel="stylesheet" href="../../vendor/css/bootstrap.min.css" />

<p style="margin-top: 20px; font-weight: bold">
  simvi - simple vi mode
</p>

<p>from tekezo's vi, dosen't work with mac os's combination move keys</p>
<p>so I added keysets like below</p>
<p>for work with mac os's combination keys</p>

<ul>
  <li>option + s + h,j,k,l</li>
  <li>shift + option + s + h,j,k,l</li>
  <li>command + s + h,j,k,l</li>
  <li>shift + command + s + h,j,k,l</li>
</ul>

<p>highly recommended to vi + mac combination keys lover users</p>
```